### PR TITLE
Allow restricting login to a specific team ID

### PR DESCRIPTION
**Pull request by bklang**

### CODE REVIEW

File: lib/ueberauth/strategy/slack.ex [hunk 1]
```diff
@@ -204,6 +207,13 @@ defmodule Ueberauth.Strategy.Slack do
     |> List.first
   end
 
+  def get_config(key) do
```
Comment (hassox) [inline lib/ueberauth/strategy/slack.ex]: The options function will pull these out of the config. for example:

```
option(conn, :team) 
```

**Comment by hassox** [comment]
I like the change :+1: but the options need to be cleaned up a little :)

File: lib/ueberauth/strategy/slack.ex [hunk 2]
```diff
@@ -31,6 +31,9 @@ defmodule Ueberauth.Strategy.Slack do
     if conn.params["state"], do: opts = Keyword.put(opts, :state, conn.params["state"])
     callback_url = callback_url(conn)
 
+    team = get_config(:team)
```
Comment (hassox) [inline lib/ueberauth/strategy/slack.ex]: You should just be able to use 

```
option(conn, :team)
```

Comment (bklang) [inline lib/ueberauth/strategy/slack.ex]: Nice, thanks. I thought something like this existed, but I couldn't find it. Fix coming up.

**Pull request merged**